etc/kayobe/kolla/config/neutron.conf

```diff
@@ -1,4 +1,4 @@
 [DEFAULT]
-dns_domain = alaskalocal.
+dns_domain = alaska.local.
```

Comment:
As discussed at the summit, need to differentiate between prod & alt-1 in the domain name.